package.json

```diff
@@ -1,15 +1,19 @@
 {
   "name": "onfido-sdk-ui",
-  "version": "0.4.0",
+  "version": "0.4.1",
   "description": "JavaScript SDK view layer for Onfido identity verification",
   "scripts": {
     "predev": "npm run build",
     "dev": "NODE_ENV=development webpack-dev-server --inline --hot --progress --https",
     "start": "superstatic build -p ${PORT:-8080} --host 0.0.0.0 --gzip -c '{\"rewrites\": [{\"source\":\"**\",\"destination\":\"index.html\"}],\"headers\":[{\"source\":\"**\",\"headers\":[{\"key\":\"Cache-Control\",\"value\":\"max-age=86400\"}]}]}'",
     "prestart": "npm run build",
-    "build": "NODE_ENV=production webpack -p --progress && npm run build:commonjs",
+    "build": "NODE_ENV=production webpack -p --progress && npm run build:commonjs && npm run build:css",
     "build:commonjs": "cross-env BABEL_ENV=commonjs babel src --out-dir lib",
+    "build:css": "cp -R src/style/ lib/style/ && lessc lib/style/style.css lib/style/style.css",
     "prebuild": "mkdir -p dist",
+    "clean": "rimraf lib dist es coverage",
```

Comment:
With the latest commit I moved some stuff to webpack. I certainly don't like long scripts inside the package.json, but I do like npm tasks. We can have a little back and forth on the next FrontEnd meeting.